src/components/Utils/Tabs.vue

```diff
@@ -137,12 +133,14 @@ export default {
     activeTab: {
       get () {
         if (!this.tabState) {
-          return this.$store.state.selectedTab
+          return this.tabs[0].hash
```

Comment:
What if there won't be any tabs?

Reply:
Then it will fail. I just tried to adjust validation to avoid that one. But if makes no sense as then it fails in Ktabs from https://kongponents.konghq.com/components/tabs.html.
I will skip it then as it's worthless to wrap it with much more code to handle that scenario, especially that we are responsible for providing the data, so it shouldn't be a case where we miss that one.

Reply:
Well, I realized that the code was not needed. 😅